### DISCUSSION TITLE
Candidates for Revisions of Measurement Unit of Length and Measurement Unit of Area

### DISCUSSION THREAD

**Discussion by michaelrabenberg** [CommonCoreOntology/CommonCoreOntologies#726] [Scope Questions]
Consider this current definition:

**Measurement Unit of Length.** A Measurement Unit that is used as a standard for measurement of one-dimensional regions or Geospatial Regions.

Two issues. (Exactly parallel issues apply to the definition of Measurement Unit of Area; I don't spell them out here to reduce clutter.)

First (less substantive) issue: I have to believe that "one-dimensional regions" and "two dimensional regions" in these definitions mean, respectively, _one-dimensional spatial regions_ and _two-dimensional spatial regions_.

Second (more substantive) issue: Either (a) a Measurement Unit of Length is used as a standard for measurement of one-dimensional spatial regions alone, or (b) that's not true. If (a), then "or Geospatial Regions" shouldn't be in the first definition, because Geospatial Regions are sites, which aren't spatial regions.

If (b), then I *assume* the right view is that a Measurement Unit of Length is used as a standard for measurement *either* of 1D spatial regions or of things that occupy 1D spatial regions. In that case too, "or Geospatial Regions" should be eliminated, because Geospatial Regions occupy three-dimenional spatial regions.

There are also more intricate possibilities: A Measurement Unit of Length is used as a standard for measurement either of one-dimensional spatial regions or of continuant parts of sites specifically, or of Geospatial Regions specifically, that occupy one-dimensional spatial regions. These too imply that the "or Geospatial Regions" is incorrect.

I guess there are other possibilities, too. I won't explore them here.

I propose the following as candidates for revised definitions:

**Length1 (option-a candidate).** A Measurement Unit that is used as a standard for measurement of one-dimensional spatial regions.

**Length2 (option-b candidate).** A Measurement Unit that is used as a standard for measurement of one-dimensional spatial regions or entities that occupy one-dimensional spatial regions.

**Length3 (option-b candidate).** A Measurement Unit that is used as a standard for measurement of one-dimensional spatial regions or continuant parts of sites that occupy one-dimensional spatial regions.

**Length4 (option-b candidate).** A Measurement Unit that is used as a standard for measurement of one-dimensional spatial regions or continuant parts of Geospatial Regions that occupy one-dimensional spatial regions.

**Comment by alanruttenberg** [comment]
So, how do any of these definitions apply to something like the length of a metal rod, or to an instruction like "if branches are longer than 2 feet then they should be cut", or "no tree is taller than 500 feet"?

**Comment by mark-jensen** [comment]
I believe we want a length measurement to be about a quality of some entity. In this case, a one-d extent that inheres in an independent continuant that isn’t a spatial region. If that’s true, then the units def should follow suit.

Sent from my iPhone

On Dec 31, 2024, at 5:54 PM, Michael Rabenberg ***@***.***> wrote:

﻿

Consider this current definition:

Measurement Unit of Length. A Measurement Unit that is used as a standard for measurement of one-dimensional regions or Geospatial Regions.

Two issues. (Exactly parallel issues apply to the definition of Measurement Unit of Area; I don't spell them out here to reduce clutter.)

First (less substantive) issue: I have to believe that "one-dimensional regions" and "two dimensional regions" in these definitions mean, respectively, one-dimensional spatial regions and two-dimensional spatial regions.

Second (more substantive) issue: Either (a) a Measurement Unit of Length is used as a standard for measurement of one-dimensional spatial regions alone, or (b) that's not true. If (a), then "or Geospatial Regions" shouldn't be in the first definition, because Geospatial Regions are sites, which aren't spatial regions.

If (b), then I assume the right view is that a Measurement Unit of Length is used as a standard for measurement either of 1D spatial regions or of things that occupy 1D spatial regions. In that case too, "or Geospatial Regions" should be eliminated, because Geospatial Regions occupy three-dimenional spatial regions.

There are also more intricate possibilities: A Measurement Unit of Length is used as a standard for measurement either of one-dimensional spatial regions or of continuant parts of sites specifically, or of Geospatial Regions specifically, that occupy one-dimensional spatial regions. These too imply that the "or Geospatial Regions" is incorrect.

I guess there are other possibilities, too. I won't explore them here.

I propose the following as candidates for revised definitions:

Length1 (option-a candidate). A Measurement Unit that is used as a standard for measurement of one-dimensional spatial regions.

Length2 (option-b candidate). A Measurement Unit that is used as a standard for measurement of one-dimensional spatial regions or entities that occupy one-dimensional spatial regions.

Length3 (option-b candidate). A Measurement Unit that is used as a standard for measurement of one-dimensional spatial regions or continuant parts of sites that occupy one-dimensional spatial regions.

Length4 (option-b candidate). A Measurement Unit that is used as a standard for measurement of one-dimensional spatial regions or continuant parts of Geospatial Regions that occupy one-dimensional spatial regions.

**Comment by michaelrabenberg** [comment]
> I believe we want a length measurement to be about a quality of some entity. In this case, a one-d extent that inheres in an independent continuant that isn’t a spatial region. If that’s true, then the units def should follow suit.

OK, so here's the current def:

**Measurement Unit of Length.** A Measurement Unit that is used as a standard for measurement of one-dimensional regions or Geospatial Regions.

Would you recommend this instead?

**Measurement Unit of Length.** A Measurement Unit that is used as a standard for measurement of One Dimensional Extents.

Similarly, here's the current def of Measurement Unit of Area:

**Measurement Unit of Area.** A Measurement Unit that is used as a standard for measurement of two-dimensional regions or Geospatial Regions.

Would you recommend this instead?

**Measurement Unit of Area.** A Measurement Unit that is used as a standard for measurement of Two Dimensional Extents.

**Comment by swartik** [comment]
@michaelrabenberg This all gets tricky.

> **Measurement Unit of Length.** A Measurement Unit that is used as a standard for measurement of One Dimensional Extents.

In the phrase "measurement of One Dimensional Extents", "measurement" is the action. So it's an Act. One participant in that Act is the entity being measured, which is an independent continuant. Another participant is the quality. The independent continuant is an input, and the quality is an output.

In the phrase "measurement of _something_", the _something_ should be the input rather than the output. I would write the definition as:

> **Measurement Unit of Length.** A Measurement Unit that is used as a standard for the One Dimensional Extent yielded by measurement of one-dimensional spatial regions or Geospatial Regions.

If that's too wordy, here's another proposal:

> **Measurement Unit of Length.** A Measurement Unit that is used as a standard for One Dimensional Extent measurements.

There's a related issue, which is that "Act of Measuring" is a CCO term. "Measurement" is not. So perhaps the definition should be:

> **Measurement Unit of Length.** A Measurement Unit that is used as a standard for One Dimensional Extents yielded by Acts of Measuring.

**Comment by alanruttenberg** [comment]
The value specification proposal is relevant here. I didn't see a response to the question regarding how to apply this to other unit-shaped entities like directives (settings) or predictions (output of computations). We don't want parallel hierarchies of units for these different use cases, which is why OBI chose value specifications as the sort of things that have units. Value specifications, a superclass of measurements, could be about any of those things. A value specification could be classified a measurement if it is the output of a measurement process, but would be other than measurements when used in different ways.

The theory (from OBI) is that measurement units are labels and aren't about anything by themselves, only in combination with a numeral.  An attempt to make that clear was to call these measurement unit *labels* in OBI. Units have history in that they are standardized via some social process. An argument might be that the units are about that standardization.

These days I think measurement units (and other things that aren't about anything by themselves) should be information structure entities. 

Related issues: #237, #104, #229, #245, #307

**Comment by oliviahobai** [comment]
@michaelrabenberg I'm converting this thread to a discussion, but feel free to open a pull request with the most recent changes to the descriptions you provided. Cheers
